bs/bs.py

```diff
@@ -400,4 +400,40 @@ def remove_codes(self, text : str):
             await ctx.send(embed = self.goodEmbed("Club info successfully edited!"))
         except KeyError:
             await ctx.send(embed = self.badEmbed(f"{key.title()} isn't saved club in this server!"))
-        
+
+    @commands.guild_only()
+    @commands.has_permissions(administrator=True)
+    async def sortroles(self, ctx):
+        for member in ctx.guild.members:
+            if discord.utils.get(member.roles, name='Guest') is not None:
+                continue
+            tag = await self.config.user(member).tag()
+            if tag is None or tag == "":
+                continue
+            try:
+                player = await self.bsapi.get_player(tag)
+
+            except brawlstats.errors.NotFoundError:
+                return await ctx.send(embed=self.badEmbed("No player with this tag found, try again!"))
+
+            except brawlstats.errors.RequestError as e:
+                return await ctx.send(embed=self.badEmbed(f"BS API is offline, please try again later! ({str(e)})"))
```

Comment:
break the for loop here probably
sending 2000 more requests when the api is down isnt a good idea